base-image/Dockerfile

```diff
@@ -184,6 +184,28 @@ ONBUILD RUN echo "Checking for pip 'requirements.txt'..." \
         ${NB_PYTHON_PREFIX}/bin/pip install --no-cache -r requirements.txt \
         ; fi
 
+ONBUILD USER root
```

Comment:
Is switching *back* to root necessary? In other words do you need the Python environment installed to successfully run install.sh?